lib/mayaUsd/render/vp2RenderDelegate/render_delegate.cpp

```diff
@@ -1024,4 +1030,14 @@ HdVP2RenderDelegate::GetSamplerState(const MHWRender::MSamplerStateDesc& desc) c
  */
 const HdVP2BBoxGeom& HdVP2RenderDelegate::GetSharedBBoxGeom() const { return *sSharedBBoxGeom; }
 
+void HdVP2RenderDelegate::CleanupMaterials()
+{
+    std::lock_guard<std::mutex> guard(_renderDelegateMutex);
```

Comment:
I don't think this function can run in parallel with HdVP2RenderDelegate ctor or dtor so we don't need to acquire this mutex here.

Reply:
Lock removed.